decentraland/components/presentational/MyAccountProgress.js

```diff
@@ -17,7 +17,8 @@ export function MyAccountProgress({ progress }) {
         borderWidth={0}
         unfilledColor={Colors.textColor}
         height={responsiveHeight(1)}
-        width={responsiveWidth(101)}
+        width={responsiveWidth(100)}
+        borderRadius={0}
```

Comment:
Oh cool! This works?

Reply:
(By "this" I mean `borderRadius`, according to our new GitHub comments rule of thumb)

Reply:
Yes it is working :+1: 

![WhatsApp Image 2019-04-16 at 6 42 22 PM](https://user-images.githubusercontent.com/29674218/56232241-30a56b00-6078-11e9-8ece-ece7135244a1.jpeg)

Reply:
Niiice. Resolving this.